client/my-sites/stats/stats-purchase/stats-commercial-upgrade-slider/index.tsx

```diff
@@ -136,9 +136,10 @@ function StatsCommercialUpgradeSlider( {
 
 	useEffect( () => {
 		// Update fetched tier quantity of the first step back to the parent component for checkout.
-		const firstStepQuantity = getTierQuentity( tiers[ 0 ] );
+		const firstStepQuantity = getTierQuantity( tiers[ 0 ] );
 		onSliderChange( firstStepQuantity as number );
 	}, [ JSON.stringify( tiers ), onSliderChange ] );
+	// TODO: Investigate if we can remove the JSON.stringify() call above.
```

Comment:
I'm not sure we need this `useEffect` block at all as we can get that value in the caller. I'll test this and push a separate PR as needed.